### ISSUE TITLE
Feature Request: Replace "Nothing Here" with "Copy Parameter Value"

### ISSUE THREAD

**Issue by jacekgajek** [MCMicS/jenkins-control-plugin#521]
## Current Behavior

After RMB on a parameter in a build view, a "Nothing here" menu is displayed
![image](https://github.com/MCMicS/jenkins-control-plugin/assets/68021034/f8679b8a-142b-4959-a6f4-6944145d10e1)

## Desired Behavior

Have some useful option - for example
- Copy parameter value
- Copy parameter name

```
IntelliJ IDEA 2023.3 EAP (Ultimate Edition)
Build #IU-233.8264.8, built on September 28, 2023
Licensed to IntelliJ IDEA EAP user: Jacek Gajek
Expiration date: October 28, 2023
Runtime version: 17.0.8.1+7-b1063.1 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 6.2.0-33-generic
Jenkins Plugin 0.13.19-2023.2
GC: G1 Young Generation, G1 Old Generation
Memory: 8048M
Cores: 24
Registry:
  debugger.new.tool.window.layout=true
  ide.experimental.ui=true
Non-Bundled Plugins:
  com.github.leomillon.uuidgenerator (4.5.1)
  de.netnexus.camelcaseplugin (3.0.12)
  IdeaVIM (2.5.1)
  org.turbanov.run.configuration.as.action (2.2)
  de.kontext_e.idea.plugins.autofill (1.2)
  com.github.stevejagodzinski.akka-log-inspection-plugin (1.2.9)
  Jenkins Control Plugin (0.13.19-2023.2)
  String Manipulation (9.11.3)
  com.jetbrains.packagesearch.intellij-plugin (233.8264.8)
  woaini.liquibase.plugin (1.3-SNAPSHOT)
  PlantUML integration (7.2.0-IJ2023.2)
  com.intellij.ml.llm (233.8264.9)
  commit-prefix-plugin (1.2.1)
  Pythonid (233.8264.8)
  com.intellij.bigdatatools.core (233.8264.8)
  org.intellij.scala (2023.3.2)
  com.rspn.git (1.9)
  mobi.hsz.idea.gitignore (4.5.1)
Kotlin: 233-1.9.0-release-358-IJ8264.8
Current Desktop: KDE
```

**Comment by MCMicS** [comment]
EAP version to test: https://plugins.jetbrains.com/plugin/6110-jenkins-control/versions/eap/429394

**Comment by MCMicS** [comment]
fixed with https://github.com/MCMicS/jenkins-control-plugin/issues/522

**Comment by MCMicS** [comment]
released with [0.13.20](https://plugins.jetbrains.com/plugin/6110-jenkins-control/versions/stable/512135)
